aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1658,4 +1658,83 @@ Tensor& diag_cpu_out(Tensor &result, const Tensor& self, int64_t dimension) {
   return result;
 }
 
+Tensor movedim(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
+  TORCH_CHECK(src.size() == dst.size(), "movedim: Invalid source or destination dims: src (",
+              src, " dims ) should contain the same number of dims as dst (", dst, "dims)");
```

Comment:
nit: you probably want a space between "dst" and "dims"

Reply:
Right. Thanks!